jobs/bump-lockfile.Jenkinsfile

```
@@ -47,65 +49,77 @@ try { lock(resource: "bump-${params.STREAM}") { timeout(time: 120, unit: 'MINUTE
     }
 
     def newPkgChecksum = shwrapCapture("jq -c .packages src/config/manifest-lock.*.json | sha256sum")
+    def newPkgTimestamp = shwrapCapture("jq -c .metadata.generated src/config/manifest-lock.*.json | xargs -I{} date --date={} +%s")
     if (newPkgChecksum == prevPkgChecksum) {
         println("No changes")
-        currentBuild.result = 'SUCCESS'
-        currentBuild.description = "[${params.STREAM}] 💤 (no change)"
-        return
+        if ((newPkgTimestamp - prevPkgTimestamp) > 172800) {
```

This would be easier to read/verify as

```suggestion
        if ((newPkgTimestamp - prevPkgTimestamp) > (2*24*60*60)) {
```